src/Domain.fs

```diff
@@ -116,7 +116,7 @@ type AbsencePolicy =
         function
         | "Holiday" -> Holiday
         | "WFH" -> WorkingFromHome
-        | "Sick" -> Sick
+        | "Sick Leave" -> Sick
```

Comment:
Can I recommend future proofing this a bit and using a regex check for "sick".

Something like this: https://github.com/mediaingenuity/Account.Validation/blob/master/src/Account.Validation/CallCredit.fs#L223C29-L223C29

In the above example `Regex` is an active pattern, with a definition of

```
let (|Regex|_|) pattern options input =
   let m = Regex.Match(input, pattern, options)
   if m.Success then Some(List.tail [ for g in m.Groups -> g.Value ])
   else None
```